lib/ddtrace/contrib/rack/middlewares.rb

```diff
@@ -55,15 +52,22 @@ def call(env)
           request_span.set_error(e)
           raise e
         ensure
+          # the source of truth in Rack is the PATH_INFO key that holds the
+          # URL for the current request; some framework may override that
+          # value, especially during exception handling and because of that
+          # we prefer using the `REQUEST_URI` if this is available.
```

Comment:
I'm OK with this, globally, but on a later release we might dig into it and figure out why a framework would do that, because maybe the `PATH_INFO`, when it's overridden, is overridden to a value that is technically less related to the "physical" http request but more meaningful in the app context.